lua/oil/buffer_cleanup.lua

```diff
@@ -0,0 +1,175 @@
+local config = require("oil.config")
+local fs = require("oil.fs")
+local util = require("oil.util")
+
+local M = {}
+
+---@class oil.BufferCleanupOptions
+---@field delete boolean
+---@field move boolean
+---@field force boolean
```

Comment:
This should be defined in the config.lua with a corresponding version where all the fields are optional to match what `setup()` expects, and the descriptions for what each of these does should be on each field.